systemtest/src/test/java/io/strimzi/systemtest/cruisecontrol/CruiseControlApiST.java

```diff
@@ -31,12 +36,34 @@
 
 @Tag(REGRESSION)
 @Tag(CRUISE_CONTROL)
+@SuiteDoc(
+    description = @Desc("This test suite verifies that Cruise Control's basic API requests function correctly"),
+    beforeTestSteps = {
+        @Step(value = "Deploy the cluster operator", expected = "Cluster operator is deployed")
+    },
+    labels = {
+        @Label(value = "cruise-control"),
```

Comment:
I just got idea -> not for this PR, as you have it already hardcoded everywhere -> but what about creating some `TestDocsLabels` where all these values will be stored - then you can just use it everywhere else instead of copy-pasting it every time.

Reply:
> I just got idea -> not for this PR, as you have it already hardcoded everywhere -> but what about creating some TestDocsLabels where all these values will be stored - then you can just use it everywhere else instead of copy-pasting it every time.

TestTags help with that idea I assume :D; Sure +1 it's great. I will create a PR with such a change.